core/src/main/java/org/frankframework/errormessageformatters/ErrorMessageFormatter.java

```diff
@@ -57,14 +63,16 @@
  *
  * @author  Gerrit van Brakel
  */
+@Log4j2
 public class ErrorMessageFormatter implements IErrorMessageFormatter, IScopeProvider {
-	protected Logger log = LogUtil.getLogger(this);
 	private final @Getter ClassLoader configurationClassLoader = Thread.currentThread().getContextClassLoader();
 
+	private @Getter @Nonnull DocumentFormat messageFormat = DocumentFormat.XML;
+
 	/**
 	 * Format the available parameters into a XML-message.
-	 *
-	 * Override this method in descender-classes to obtain the required behaviour.
+	 * <b/>
```

Comment:
self closing bold tag? ;-) 

```suggestion
	 * <br/>
```